README.rst

```diff
@@ -143,6 +143,7 @@ Oceania
 Africa
 ------
 
+* Angola
```

Comment:
Please swap this one with Algeria, to keep the alphabetical order.

Reply:
:+1: